contract-map.json

```diff
@@ -1,4 +1,18 @@
 {
+  "0x106b419718298f91ca576728A670597fb2e0eE4e": {
+    "name": "Genevieve exchange",
+    "logo": "gxe.png",
+    "erc20": true,
+    "symbol": "GXE",
+    "decimals": 0
```

Comment:
On the live contract, it says 10 decimals. This is not consistent. Can you clarify why this is correct?